### PR TITLE
BD-57 feat: add alert component

### PR DESCRIPTION
**Pull request by SlavaMryasov**

### CODE REVIEW

File: src/components/alert/Alert.tsx
Comment (sfgardev) [inline src/components/alert/Alert.tsx:1]: forgot to add re-exports

File: src/components/alert/Alert.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+export const Alert = forwardRef<HTMLDivElement, AlertProps>(
+  ({ className, isOpened, message, onClose, type }, ref) => {
+    if (!isOpened) {
+      return null
+    }
+
+    return (
+      <div
+        className={cn(
+          'flex h-12 w-[387px] flex-row justify-between border px-[24px] pb-[12px] pt-[12px] text-light-100',
```
Comment (sfgardev) [inline src/components/alert/Alert.tsx:22]: 1.remove height
image

2.do not use arbitrary values if possible

File: src/components/alert/Alert.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          {
+            'border-danger-500 bg-danger-900': type === 'error',
+            'border-success-500 bg-success-900': type === 'success',
+            'border-warning-500 bg-warning-900': type === 'warning',
+            className,
```
Comment (sfgardev) [inline src/components/alert/Alert.tsx:27]: replace className from object

File: src/components/alert/Alert.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          }
+        )}
+        ref={ref}
+      >
+        <span>
+          {type === 'error' && <b>Error! </b>}
```
Comment (sfgardev) [inline src/components/alert/Alert.tsx:33]: like in design, but looks strange, we should discuss it

**Review by sfgardev — CHANGES_REQUESTED** [review]

File: src/components/alert/Alert.tsx
Comment (SlavaMryasov) [reply src/components/alert/Alert.tsx:1]: done

File: src/components/alert/Alert.tsx (hunk 1 above)
Comment (SlavaMryasov) [reply src/components/alert/Alert.tsx:22]: done

File: src/components/alert/Alert.tsx (hunk 2 above)
Comment (SlavaMryasov) [reply src/components/alert/Alert.tsx:27]: done

File: src/components/alert/Alert.tsx (hunk 3 above)
Comment (SlavaMryasov) [reply src/components/alert/Alert.tsx:33]: done

**Review by sfgardev — APPROVED** [review]

**Pull request merged**